src/i18n/en-US.json

```diff
@@ -1501,6 +1501,11 @@
   "teamCreationSuccessTitle": "Congratulations {{name}}!",
   "teamCreationTitle": "Create your team",
   "teamCreationUserNotFoundError": "User not found",
+  "teamCreationLeaveModalCloseLabel": "Close window Leave without saving",
```

Comment:
punctuation missing?

Reply:
Not in Figma, should I add?

Reply:
just looked at figma, it's probably fine for the acc string, mostly intended at screen reader.
Let merge :rocket: